### PR TITLE
➖ Remove deps lodash.debounce

### PR DESCRIPTION
**Pull request by nkokla**

### CODE REVIEW

File: src/utils/debounce.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const debounce = (handler: (...arg: any) => any, timer: number) => {
+  let _timeOut: NodeJS.Timeout;
```
Comment (desoindx) [inline src/utils/debounce.ts:2]: why an _ ?

File: src/components/addressAutocomplete/useSuggestions.tsx [hunk 2]
```diff
@@ -17,6 +17,20 @@ type configProps = {
   minCharactersLength?: number;
 };
 
+const debounce = (handler: (...arg: any) => any, timer: number) => {
```
Comment (desoindx) [inline src/components/addressAutocomplete/useSuggestions.tsx:20]: why dont you use your service ?

Comment (nkokla) [reply src/components/addressAutocomplete/useSuggestions.tsx:20]: It is an work in progress. I have made the service here before extract but this function won't stay here.

Comment (nkokla) [reply src/components/addressAutocomplete/useSuggestions.tsx:20]: Why not :-p Sometimes i use this syntax for the uninterestings let/var, but if you prefer, i clean.

File: src/utils/debounce.ts (hunk 1 above)
Comment (nkokla) [reply src/utils/debounce.ts:2]: Why not :-p Sometimes i use this syntax for the uninterestings let/var, but if you prefer, i clean.

File: src/components/addressAutocomplete/useSuggestions.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import debounce from 'lodash.debounce';
+// import debounce from 'lodash.debounce';
```
Comment (desoindx) [inline src/components/addressAutocomplete/useSuggestions.tsx:1]: Still a comment here

File: src/components/addressAutocomplete/useSuggestions.tsx (hunk 2 above)
Comment (desoindx) [reply src/components/addressAutocomplete/useSuggestions.tsx]: then don't ask me to review it...

**Review by desoindx — APPROVED** [review]

**Pull request merged**